unittests.py

```diff
@@ -1,15 +1,34 @@
 #!/usr/bin/env python
 
 import unittest
+import doctest
 import pynance.dummy_test
 import pynance.textimporter_test
 
+def doc_test_suite():
+    "Returns the testsuite doctests for all modules. Please don't forget to add new modules here."
```

Comment:
I think it's rather certain, that we will forget this at some point 😃 
I would suggest we make sure that all modules listed in the `__init__.py` as `__all__` should be added for the test automatically.

Reply:
That would probably be another PR, but we should think about just using the auto-discovery of `pytest`

Reply:
This is a great idea! I don't think that this is another PR, I'll try to fit it into this one here now.

Reply:
I've added code that will automatically discover all submodules in fc1f321d4417437776dca3042ef2747f916c02a6 in order to run `doctest` on them.

Reply:
nice solution!